plugins/pipeline_plugin/transform/default_transform.py

```diff
@@ -0,0 +1,25 @@
+from pipeline_plugin.transform.general_transformers.cod_transform import transform_cod
+from pipeline_plugin.transform.general_transformers.osm_transform import transform_osm
+from pipeline_plugin.transform.general_transformers.transform_to_crs import (
+    transform_to_crs,
+)
+from pipeline_plugin.utils.files import load_file, save_shapefiles
+
+
+def default_transform(
```

Comment:
I don't think this is an ideal name. It isn't really a "default" transform as it contains source-specific behaviour. Though I'm not sure what a better name would be.